lib/modules/neovim.nix

```diff
@@ -17,10 +17,14 @@ let
       username = "nobody";
     in
     home-manager.lib.homeManagerConfiguration {
-      inherit system username;
-      configuration = { };
-      homeDirectory = "/home/${username}";
       pkgs = builtins.getAttr system nixpkgs.outputs.legacyPackages;
+      modules = [{
+        home = {
+          inherit username;
+          stateVersion = "22.11";
```

Comment:
Should it be up to the user to specify this?

Reply:
This function is an implementation detail of [themes](https://github.com/SoxinOS/soxin/blob/35699f045f5bc07cc8a4378043387cc25ab9ceb6/lib/modules/themes.nix#L71-L123) in Soxin. I don't think this case was addressed in its design. It's hardcoded, but it should probably be set to some external version number coming from home-manager itself. I don't see a way to get that, do you know of a way?

Reply:
If we define a current supported version in soxin, which is what we were discussing doing with branch-offs, we can probably get that value.

Reply:
Or we can leave it hardcoded, stateVersion probably isn't going to affect the neovim module, which is the only thing we are interested in in this particular case.

Reply:
I think for now we can leave it hardcoded, and when we explore our own stateVersion we can think of it for this module but yea I doubt it'll have an effect on NeoVim.